### ISSUE TITLE
RUN - archivo 1

### ISSUE THREAD

**Issue by crist0balsoto** [IIC2413/Syllabus-2024-2#28]
Hola, tengo una pregunta y es que en el archivo 1 todos los run son el mismo string, lo cual nose si es un error o algo por el estilo.
![image](https://github.com/user-attachments/assets/e4704f07-25ac-4117-b632-87f120d862a0)

**Comment by tom4stapia** [comment]
Hola! No es un error. Todas esas filas corresponden a la misma persona que ha tomado diferentes cursos. Puedes notar la diferencia a partir de la columna "Periodo curso" en el archivo.

**Comment by Utmite** [comment]
O sea que se cambió miles de veces el nombre?

**Comment by Utmite** [comment]
O sea que se cambió miles de veces el nombre?

**Comment by tom4stapia** [comment]
Efectivamente, hay un error con los RUN. Se subirá un nuevo archivo a la brevedad.
